Real-Time Corruptor/BizHawk_RTC/BizHawk.Client.EmuHawk/RTC/RTC_Extensions.cs

```diff
@@ -644,16 +644,18 @@ public static List<Control> getControlsWithTag(this Control.ControlCollection co
 		#endregion CONTROL EXTENSIONS
 
 		#region PATH EXTENSIONS
-
+		//why not just use 
```

Comment:
No idea. Might as well change it over. Just ensure that it appends the trailing backslash

Reply:
I will change this on a seperate request, there are several places where this is used that will have to be checked for correctness